README.md

```diff
@@ -138,6 +120,38 @@ Notable features:
 [^5b]: https://www.ubicloud.com/blog/ubicloud-hosted-arm-runners-100x-better-price-performance
 [^5c]: https://github.com/ubicloud/ubicloud/blob/main/routes/web/webhook/github.rb
 
+### RunsOn
+
+[RunsOn](https://runs-on.com/) is a self-hosted GitHub Actions runner tool that runs on your own AWS account. It's **free for non-commercial usage**, and comes at a €300 flat yearly fee for commercial projects.
+
+Pricing [here](https://runs-on.com/).
+
+Notable features:
+- **Easy install with a QuickFormation template** (video guide available).
+- Cheap, flat-rate pricing with an easy-to-use [pricing calculator](https://runs-on.com/calculator/) that you can use to calculate your spending.
+- ARM & GPU support.
```

Comment:
X64, ARM & GPU support

Reply:
I'm omitting X86-64 as that's something that all providers currently support, although this is good feedback and it arguably should be made more explicit in a future PR.